inst/tests/tests.Rraw

```diff
@@ -18814,3 +18814,7 @@ test(2238.6, "a" %notin% integer(), TRUE)
 test(2238.7, "a" %notin% NULL, TRUE)
 test(2238.8, NA %notin% 1:5, TRUE)
 test(2238.9, NA %notin% c(1:5, NA), FALSE)
+
+# rbindlist segfault with fill=TRUE and usenames=FALSE #5444
```

Comment:
if this is fixing an issue caused by a change in dev, maybe the tests should be attached to the same suite as the original PR?

Reply:
Moved them accordingly